thunder/executors/nvfuserex_impl.py

```diff
@@ -3188,6 +3181,51 @@ def argsort_transform(
 # Register argsort with NVFuser
 register_supported(prims.argsort, argsort_transform, _argsort_check_)
 
+
+def _cumsum_check(a: TensorProxy, dim: int, /, dtype: dtypes.dtype | None = None) -> bool:
+    if a.ndim != 1:
```

Comment:
We should document (maybe add a TODO) that this lowering needs to be updated once nvfuser supports cumsum